index.js

```diff
@@ -13,22 +13,19 @@ let attempts = 0;
 const maxNumberOfAttempts = 5;
 
 // Returns a random number from min (inclusive) to max (exclusive)
-// Usage:
-// > getRandomNumber(1, 50)
-// <- 32
-// > getRandomNumber(1, 50)
-// <- 11
```

Comment:
These comments probably could have stayed. They're just describing how to use the `getRandomNumber()` function.